server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt

```diff
@@ -122,6 +122,7 @@ class KotlinLanguageServer : LanguageServer, LanguageClientAware, Closeable {
                 sourcePath.refresh()
             }
         }
+		progress?.close()
```

Comment:
Please fix the indentation here.

Reply:
Sorry about that. kotlin-mode in Emacs forces tabs as the default, and editorconfig support is not perfect. Hope to have less of these issues when #342 is in place... So easy to forget 😿